app/views/account/activations/new.html.erb

```diff
@@ -0,0 +1,108 @@
+<div class='col-md-8'>
+  <div class='card mx-4'>
+    <div class='card-body p-4'>
+      <h1 class='text-center'><%= t('.activation') %></h1>
+      <p class='text-muted text-center'><%= t('.account_activation') %></p>
+      <%= simple_form_for(:activation, url: activation_path, html: { method: 'patch' }) do |f| %>
```

Comment:
Burada update yaptığımız için `PATCH` seçtin sanırım. Ancak bu RESTful olmamış. `PATCH` yolunu bildiğimiz tekil bir resource'a yapılır, örneğin `activation/123` gibi. Eğer resource'u bilmiyorsak ve `/activation`'a istek atıyorsak REST bakış açısıyla bu metod `POST` olmalı.